README.md

```diff
@@ -1,3 +1,25 @@
+# DIANNA: Deep Insight And Neural Network Analysis
+
+## How to use dianna
+
+The project setup is documented in [project_setup.md](project_setup.md). Feel free to remove this document (and/or the link to this document) if you don't need it.
+
+## Installation
+
+To install dianna directly from GitHub repository, do:
```

Comment:
There's a missing "the" here:  "from the GitHub repository"

Reply:
```suggestion
To install dianna directly from the GitHub repository, do:
```

Reply:
Can you commit that suggestion? I can't :D Otherwise, maybe if you make the suggestion I can merge it...

Reply:
Oh I didn't know that was a feature! Committed and I'll merge it